### DISCUSSION TITLE
Unable to re-add Zooz Zen15 device

### DISCUSSION THREAD

**Discussion by jsbrich** [zwave-js/zwave-js#7436] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Exclude working ZEN15 device while trying to do OTA update.  Update never started, found there may be a bug, so went to re-add device.  Inclusion works, but won't finish interview.  In log I find
`2024-11-22T16:42:09.474Z CONFIG   Writing the device index to disk failed: ENOENT: no such file or directory, op
                                  en '/snapshot/build/node_modules/@zwave-js/config/config/devices/index.json'
2024-11-22T16:45:32.015Z CNTRLR   [Node 025] Error during node interview: Argument validation failed:
                                  Expected parameter allowCache to be a boolean, got undefined (ZW0322)`
Node 025 is what I added.  First line would be able the time I started Inclusion, not sure if it is related or not.  I get the same result if I use normal inclusion or smart scan.

What did you expect to happen instead?
Device to be added and show options correctly.

  1. Include ZEN15 device

Anything else we should know?

### Software versions

Driver (node-zwave-js): ...14.3.5

Z-Wave JS UI: ... 9.27.6

Home Assistant Z-Wave Integration: ...

Home Assistant Z-Wave JS Addon: ...

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Z-Wave Controller (Stick/Dongle/...)

Zooz ZST39 LR 800 Series

### Device information

Manufacturer: ...Zooz
Model name: ...Zen15
Node ID: ...025

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

2024-11-22T17:01:49.528Z CONFIG   Writing the device index to disk failed: ENOENT: no such file or directory, op
                                  en '/snapshot/build/node_modules/@zwave-js/config/config/devices/index.json'
2024-11-22T17:03:25.602Z CNTRLR   [Node 025] Error during node interview: Argument validation failed:
                                  Expected parameter allowCache to be a boolean, got undefined (ZW0322)
2024-11-22T17:03:25.839Z CNTRLR   [Node 025] Error during node interview: Argument validation failed:
                                  Expected parameter allowCache to be a boolean, got undefined (ZW0322)

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment, marked as answer]
Fixed in the upcoming release 14.3.6 / Z-UI 9.27.7. I'm working on releasing both right now.

**Reply by jsbrich** [reply]
I just got the notification of an update.  After the update, it was able to finish the interview and appears to be showing correctly.  Thanks
